.gitignore

```diff
@@ -40,4 +40,5 @@ jspm_packages
 .next
 .idea
 
-package-lock.json
\ No newline at end of file
+package-lock.json
+nteract-members.json
```

Comment:
Could we put this in `generated/nteract-members.json`?

Reply:
👍 Moved